docs/next/guides/building-and-testing/plugins.md

```diff
@@ -1,242 +1,175 @@
     this.addHook('afterChange', (changes, source) => this.onAfterChange(changes, source));
 
-    // The super method assigns the this.enabled property to true, which can be later used to check if plugin is already enabled.
+    // The super method assigns the this.enabled property to true.
+    // It is a necessary step to update the plugin's settings properly.
     super.enablePlugin();
   }
 
   /**
-   * The disablePlugin method is used to disable the plugin. Reset all of your classes properties to their default values here.
+   * The disablePlugin method is used to disable the plugin.
    */
   disablePlugin() {
-    this.yourProperty = '';
-    this.anotherProperty = [];
+    // Reset all of your plugin class properties to their default values here.
+    this.configuration = null;
 
     // The super method takes care of clearing the hook connections and assigning the 'false' value to the 'this.enabled' property.
     super.disablePlugin();
   }
 
   /**
-   * The updatePlugin method is called on the afterUpdateSettings hook (unless the updateSettings method turned the plugin off).
-   * It should contain all the stuff your plugin needs to do to work properly after the Handsontable instance settings were modified.
+   * The updatePlugin method is called on the `afterUpdateSettings` hook
+   * (unless the updateSettings method turned the plugin off).
+   *
+   * It should contain all the stuff your plugin needs to do to work correctly
+   * after updating Handsontable instance settings.
    */
   updatePlugin() {
-
-    // The updatePlugin method needs to contain all the code needed to properly re-enable the plugin. In most cases simply disabling and enabling the plugin should do the trick.
+    // The `updatePlugin` method needs to contain all the code needed to properly re-enable the plugin.
+    // In most cases simply disabling and enabling the plugin should do the trick.
     this.disablePlugin();
     this.enablePlugin();
 
     super.updatePlugin();
   }
 
+  /**
+   * Define your external methods.
+   */
+  externalMethodExample() {
+    // Method definition.
+  }
+
   /**
    * The afterChange hook callback.
    *
-   * @param {Array} changes Array of changes.
-   * @param {String} source Describes the source of the change.
+   * @param {CellChange[]} changes An array of changes.
+   * @param {string} source Describes the source of the change.
    */
   onAfterChange(changes, source) {
     // afterChange callback goes here.
   }
 
   /**
-   * The destroy method should de-assign all of your properties.
+   * The `destroy` method is the best place to clean up any instances,
+   * objects or index mappers created during the plugin's lifecycle.
    */
   destroy() {
-    // The super method takes care of de-assigning the event callbacks, plugin hooks and clearing all the plugin properties.
+    // The super method takes care of de-assigning the event callbacks,
```

Comment:
The super method -> This method

Reply:
The original text is referring to the nature of class inheritance.
Running `super.xyz()` in the child class will invoke the `xyz` method from the parent class (simplified description).
Although, sure, we can mention that's the `BasePlugin.disablePlugin` method.